src/core/messages.js

```diff
@@ -1,27 +1,27 @@
 const messages = [
-  "Nice job",
-  "Looks good",
-  "Clean fix!",
-  "Well done",
-  "Thank you! Really nice PR",
-  "Great job",
-  "Looks amazing",
+  'Nice job',
+  'Looks good',
+  'Clean fix!',
+  'Well done',
+  'Thank you! Really nice PR',
+  'Great job',
+  'Looks amazing',
   "Thank you! I'm so happy that we finally did this",
-  "Looks nice!",
-  "Nice refactor!",
+  'Looks nice!',
+  'Nice refactor!',
   "Amazing Job 🥇 It was a complex task you've pulled off!",
-  "Great work on this!",
-  "Looking good so far",
-  "I think this is a good change",
-  "This is looking",
-  "Superb work! You completed a challenging task",
-  "Excellent work",
-  "This is an excellent improvement",
-  "Nice and tidy, great job!",
-  "Outstanding job!",
-  "Nicely done!",
-  "Way to go!",
-  "Good feature, let's merge it!"
+  'Great work on this!',
+  'Looking good so far',
+  'I think this is a good change',
+  'This is looking',
+  'Superb work! You completed a challenging task',
+  'Excellent work',
+  'This is an excellent improvement',
+  'Nice and tidy, great job!',
+  'Outstanding job!',
+  'Nicely done!',
+  'Way to go!',
+  "Good feature, let's merge it!",
 ];
 
-export { messages };
+export default messages;
```

Comment:
Can we ignore this rule? From my experience default export always makes issues with the naming when someone use it like `import messagesList from 'messages';`. Sometimes it's harder to keep in mind that messagesList it's just messages and it's harder to refactor that